api/spanner/service/app.yaml

```diff
@@ -0,0 +1,24 @@
+runtime: custom
+env: flex
+service: spannerwrite
+
+# Should match something at https://cloud.google.com/compute/docs/machine-types.
+resources:
+  # g1-small
+  cpu: 0.5
+  memory_gb: 1.70
+
+# Setting values under automatic_scaling yields error:
```

Comment:
What values did you try to set?

Reply:
[These two](https://github.com/web-platform-tests/wpt.fyi/pull/635/commits/3f92a83de8f035deaf47fbdb2bd20394b22c1432)